sonic_y_cable/y_cable.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
+#
+# y_cable.py
+#
+#  class for implementing Y cable access and configurations
+#   API's for Y cable functionality in SONiC
+
+try:
+    import binascii
+    import sys
+
+    import sonic_platform.platform
+    import sonic_platform_base.sonic_sfp.sfputilhelper
+
+    from sonic_py_common import logger
+
+except ImportError as e:
+    raise ImportError(str(e) + " - required module not found")
+
+
+PLATFORM_SPECIFIC_MODULE_NAME = "sfputil"
+PLATFORM_SPECIFIC_CLASS_NAME = "SfpUtil"
```

Comment:
These reference the old platform plugins, and you are not using them. Should remove.